commercetools-models/src/main/java/io/sphere/sdk/carts/expansion/ShippingRateInputExpansionModel.java

```diff
@@ -0,0 +1,4 @@
+package io.sphere.sdk.carts.expansion;
+
+public interface ShippingRateInputExpansionModel {
```

Comment:
Is an empty exapnsion model really useful?

Reply:
Actually this should be removed 👍